conf/gunicorn/app.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+bind = "unix:/tmp/gunicorn-web.sock"
+proc_name = "web"
```

Comment:
I'm not really sure why we bind to UNIX sockets instead of just a host and port in production. It was introduced by a previous PR. I've left it the same here.

Also not really sure what `proc_name` actually does but I've just left it the same.